linux_os/guide/services/fapolicyd/fapolicyd_prevent_home_folder_access/rule.yml

```diff
@@ -21,21 +21,11 @@ references:
     nist: CM-6 b
     srg: SRG-OS-000480-GPOS-00230
 
-ocil_clause: 'fapolicyd allows non-privileged users to grant other users direct access to the contents of their home directories/folders'
+ocil_clause: 'This rule has been deprecated.'
 
 ocil: |-
-    Verify that fapolicyd on {{{ full_name }}} prevents ability of non-privileged users to grant other users direct access to the contents of their home directories/folders.
+    This rule has been deprecated.
 
-    Run the following command:
-
-    grep -r "deny_audit perm=chmod path=/home" /etc/fapolicyd/rules.d
-
-fixtext: |-
-    Configure fapolicyd to ability of non-privileged users to grant other users direct access to the contents of their home directories/folders.
-    Add or edit the following lines in /etc/fapolicyd/rules.d/90-deny-home.
-
-    deny_audit perm=chmod path=/home all : all
-
-    Note: That fapolicyd must have correctly configured rules. All configurations will be based on the actual system setup and organizational polices and practices.
-
-srg_requirement: '{{{ full_name }}} must limit the ability of non-privileged users to grant other users direct access to the contents of their home directories/folders.'
+warnings:
+    - general: |-
+          This rule is deprecated and there is no replacement at this time.
```

Comment:
I need some explanation - there is no reason mentioned here or in the commit messages or in the PR description. You wrote that it's a "bad advice" but I'm curious why.

Reply:
If you apply this rule fapolicyd will not start.

Reply:
@Mab879 so please add it to the warning text